roles/sap_ha_install_pacemaker/README.md

```diff
@@ -1,135 +1,150 @@
 # sap_ha_install_pacemaker Ansible Role
 
-Ansible role for SAP Pacemaker Setup
+Ansible role for SAP Pacemaker Setup. This role needs **sap_ha_prepare_pacemaker** to be able to run. It creates a 2 node pacemaker cluster.
 
 ## Scope
 
 - **RedHat Enterprise Linux**
-    - Tested on RHEL 8.2
+    - Tested on RHEL 8.2, 8.4 and later
+    - Please also check 
+    - [Overview of the Red Hat Enterprise Linux for SAP Solutions subscription](https://access.redhat.com/solutions/3082481)
+    - [Automating SAP HANA Scale-Up System Replication using the RHEL HA Add-On](https://access.redhat.com/articles/3004101)
 
-- **Azure** 
+- **Azure**
     - Tested
     - Followed the steps based on the guide published in
         - [Azure Pacemaker Setup Guide](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-rhel-pacemaker)
 
 - **AWS**
```

Comment:
Would suggest defining the IaaS type, e.g. `, EC2 Instance`

Also to control scope, add the `AWS Classic environment` is out of scope?

Reply:
As discussed over Slack, I was referring to clarity in the IaaS Types from each Cloud Service Provider (aka. hyperscaler) and was suggesting a suffix to the header. The approach taken in the README now is more concise and clearer, resolved by 3218e06